product_docs/docs/pem/8/registering_database_server.mdx

```diff
@@ -8,7 +8,153 @@ redirects:
+| `--team`                | Specifies the name of the group role that's allowed to access the server.                                                                                                                                                                                                                                                                                                                                                                                                |
+| `--owner`               | Specifies the name of the role that owns the monitored server.                                                                                                                                                                                                                                                                                                                                                                                                             |
+| `--display-name <name>` | Specifies the display name of the monitored database server. The default is the system hostname.                                                                                                                                                                                                                                                                                                                                                                                   |
+| `--config-dir`          | Specifies the directory path of the agent configuration file. The default is `"<pemworker path>/../etc"`.                                                                                                                                                                                                                                                                                                                                                                   |
+
+Set the environment variable `PEM_SERVER_PASSWORD` to provide the password for the PEM server to allow the pemworker to connect as a PEM admin user.
+
+Set the environment variable `PEM_MONITORED_SERVER_PASSWORD` to provide the password of the database server being registered and monitored by pemagent.
+
+If you don't provide the password, a password authentication error occurs. The PEM server acknowledges that the server was registered properly.
+
+#### Examples
+
+The following example registers a server using only the required parameters. 
+The user `admin01` will be used to connect to the PEM server and the credentials supplied for the `--server-*` parameters will be used to create a connection from the agent to the monitored server.
```

Comment:
```suggestion
The user `admin01` connects to the PEM server, and the credentials supplied for the `--server—*` parameters create a connection from the agent to the monitored server.
```

Reply:
I get (again) that this is house style. But collapsing this into the present tense make it feel vague to me. The future tense implies '_when_ the user presses enter on this command something _will_ happen'. The present tense makes it ambiguous whether we are defining what `admin01` is, how `admin01` is typically used, or how it _will_ be used in this concrete case.